src/main/kotlin/stringcalculator/Calculator.kt

```diff
@@ -0,0 +1,49 @@
+package stringcalculator
+
+import java.lang.RuntimeException
+import kotlin.NumberFormatException
+
+class Calculator(private val input: String?) {
+
+    private fun numberValidate(str: String): Int {
+        return try {
+            if (str.toInt() < 0) throw RuntimeException(NOT_NATURAL_NUMBER_ERROR_MESSAGE)
+            str.toInt()
+        } catch (e: NumberFormatException) {
+            throw RuntimeException(NOT_NATURAL_NUMBER_ERROR_MESSAGE)
+        }
```

Comment:
```suggestion
        val number = str.toIntOrNull() 
        if (number == null || number < 0) throw RuntimeException(NOT_NATURAL_NUMBER_ERROR_MESSAGE)
        return number
```
[toIntOrNull](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-int-or-null.html) 함수를 활용하면 다음처럼 작성할 수 있습니다 🙂